json/src/fs2/data/json/package.scala

```diff
@@ -58,6 +58,66 @@ package object json {
   def values[F[_], Json](implicit F: RaiseThrowable[F], builder: Builder[Json]): Pipe[F, Token, Json] =
     ValueParser.pipe[F, Json]
 
+  /** Transforms a stream of Json values into a stream of Json tokens.
+    *
+    * This operation is the opposite of `values`.
+    */
+  def tokenize[F[_], Json](implicit tokenizer: Tokenizer[Json]): Pipe[F, Json, Token] =
+    _.flatMap(value => Stream.emits(tokenizer.tokenize(value).toList))
+
+  /** Json Token stream pipes to render Json values. */
+  object render {
+
+    /** Renders a compact representation of the Json token stream.
+      *
+      * Chunks can be concatenated to render all values in the stream,
+      * separated by new lines.
+      *
+      * You can use this to write the Json stream to a file.
+      */
+    def compact[F[_]]: Pipe[F, Token, String] =
+      Renderer.pipe[F](false, "")
+
+    /** Renders a pretty-printed representation of the token stream with the given
+      * indentation size.
+      *
+      * Chunks can be concatenated to render all values in the stream,
+      * separated by new lines.
+      *
+      * You can use this to write the Json stream to a file.
+      */
+    def pretty[F[_]](indent: Int = 2): Pipe[F, Token, String] =
```

Comment:
Both for here and the collector: Maybe let's not choose a side on the spaces vs tabs war and allow to specify the indentation string as another parameter with default? Maybe also the newline? No need to reproduce the full power of circe's pretty printer or similar, but these options seem very straight-forward to add…

Reply:
:sob: Ok…